### ISSUE TITLE
Add A11y

### ISSUE THREAD

**Issue by a1ly404** [a1ly404/derby-stat-tracker#2]
**Is your feature request related to a problem? Please describe.**
Yes the website needs to be accessible 

**Describe the solution you'd like**
Update the repo to include the a11y feedback from vercel

- ARIA hidden element must not be focusable or contain focusable elements
   - Ensures aria-hidden elements are not focusable nor contain focusable elements [Learn More](https://dequeuniversity.com/rules/axe/4.9/aria-hidden-focus?application=Vercel%20Toolbar)

- Document should have one main landmark
   - Ensures the document has a main landmark [Learn More](https://dequeuniversity.com/rules/axe/4.9/landmark-one-main?application=Vercel%20Toolbar)
